docs/content/docs/tutorials/sagemaker-example.md

```diff
@@ -0,0 +1,362 @@
+export IAM_ROLE_ARN_FOR_IRSA=$(aws --region $AWS_DEFAULT_REGION iam get-role --role-name $OIDC_ROLE_NAME --output text --query 'Role.Arn')
+echo $IAM_ROLE_ARN_FOR_IRSA
+```
+
+For more information on authorization and access for ACK service controllers, including details regarding recommended IAM policies, see [Configure Permissions][configure-permissions].
+
+### Install the SageMaker ACK service controller
+
+Get the SageMaker Helm chart and make it available on the `Deployment` host with the following commands:
+
+```bash
+export HELM_EXPERIMENTAL_OCI=1
+export SERVICE=sagemaker
+export RELEASE_VERSION=v0.0.4
+export CHART_EXPORT_PATH=/tmp/chart
+export CHART_REPO=public.ecr.aws/aws-controllers-k8s/$SERVICE-chart
+export CHART_REF=$CHART_REPO:$RELEASE_VERSION
+export ACK_K8S_NAMESPACE=ack-system
+
+mkdir -p $CHART_EXPORT_PATH
+helm chart pull $CHART_REF
+helm chart list
+helm chart export $CHART_REF --destination $CHART_EXPORT_PATH
+```
+
+Update the Helm chart values for a cluster-scoped `Deployment`. 
+
+```bash
+# Update the following values in the Helm chart
+cd $CHART_EXPORT_PATH/$SERVICE-chart
+yq e '.aws.region = env(AWS_DEFAULT_REGION)' -i values.yaml
+yq e '.aws.account_id = env(AWS_ACCOUNT_ID)' -i values.yaml
+yq e '.serviceAccount.annotations."eks.amazonaws.com/role-arn" = env(IAM_ROLE_ARN_FOR_IRSA)' -i values.yaml
+cd -
+```
+
+Install the relevant custom resource definitions (CRDs) for the SageMaker ACK service controller. 
+
+```bash
+kubectl apply -f $CHART_EXPORT_PATH/$SERVICE-chart/crds
+```
+
+Create a namespace and install the SageMaker ACK service controller with the Helm chart. 
+```bash
+helm install -n $ACK_K8S_NAMESPACE --create-namespace --skip-crds ack-$SERVICE-controller \
+ $CHART_EXPORT_PATH/$SERVICE-chart
+```
+
+Verify that the CRDs and Helm charts were deployed with the following commands:
+```bash
+kubectl get crds
+kubectl get pods -n $ACK_K8S_NAMESPACE
+```
+
+## Train an XGBoost model
+
+### Prepare your data
+
+For training a model with SageMaker, we will need an S3 bucket to store the dataset and model training artifacts. For this example, we will use [MNIST](http://yann.lecun.com/exdb/mnist/) data stored in [LIBSVM](https://www.csie.ntu.edu.tw/~cjlin/libsvm/) format.
+
+First, create a variable for the S3 bucket:
+
+```bash
+export RANDOM_VAR=$RANDOM
+# Specify your service region
+export SERVICE_REGION=us-east-1
+export SAGEMAKER_BUCKET=ack-sagemaker-bucket-$RANDOM_VAR
+```
+
+Then, create a file named `create-bucket.sh` with the following code block:
+
+```bash
+printf '
+#!/usr/bin/env bash
+# Create the S3 bucket
+if [[ $SERVICE_REGION != "us-east-1" ]]; then
+  aws s3api create-bucket --bucket "$SAGEMAKER_BUCKET" --region "$SERVICE_REGION" --create-bucket-configuration LocationConstraint="$SERVICE_REGION"
+else
+  aws s3api create-bucket --bucket "$SAGEMAKER_BUCKET" --region "$SERVICE_REGION"
+' > ./create-bucket.sh
+```
```

Comment:
certainly not required for this PR, but it would be awesome in the future to have a tutorial that used the S3 ACK controller and an [S3 Bucket CR](https://aws-controllers-k8s.github.io/community/reference/s3/v1alpha1/bucket/) instead of calling the `aws s3api create-bucket` command :)

Reply:
I agree! Adding to To-do's :)